hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleNetworkContext.java

```diff
@@ -504,6 +504,7 @@ public Hash getHash() {
         try (final var out = new SerializableDataOutputStream(baos)) {
             serializeNonHashData(out);
             out.write(blockHashes.getHash().getValue());
```

Comment:
Should we be deleting line 506 ?

Reply:
+1.

Perhaps we could simply nuke the `getValue()` method altogether - this would immediately reveal all the usages and help fix all of them at once w/o having to introduce a `@Deprecated` method.

Reply:
@anthony-swirldslabs its too many changes to put into a single PR. The next PR after this one should finally remove it. And then probably another to remove the `ImmutableHash` class.

Reply:
@Neeharika-Sompalli you are right, my mistake. Fixed now

Reply:
Sounds good to me.